chain-abci/src/app/macros.rs

```diff
@@ -0,0 +1,48 @@
+// Fight borrow checker
```

Comment:
not that useful comment -- I assume these are for buffer constructor boilerplate? Maybe better to put them in chain-storage + more explanatory comment?

Reply:
They are closely relate to implementation detail of `ChainNodeApp` (the field names), ideally it should be methods of `ChainNodeApp`, but borrow checker won't allow that, because it will retain the whole reference to `self`, although it only need several fields.

Reply:
Added some comments there.